std/concurrency.d

```diff
@@ -1422,57 +1333,56 @@ private:
         void switchContext() nothrow
         {
             mutex_nothrow.unlock_nothrow();
-            scope(exit) mutex_nothrow.lock_nothrow();
+            scope (exit) mutex_nothrow.lock_nothrow();
```

Comment:
Same line if one liner. Apply other rules for block statements to `scope` when there are multiple lines.